docs/aurora-cloud/chain.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Dedicated Blockchains"
+sidebar_position: 1
+---
+
+
+# Aurora Chains
+
+Aurora Chains are dedicated EVM-compatible blockchains built on top of the NEAR Protocol. They are your own instance of Aurora – every Aurora Chain is based upon the Aurora smart contract,
+also called [Aurora Engine](/getting-started/aurora-engine). It provides you with a customized and dedicated blockchain environment.
+
+Benefits include:
+
+1. *No need for additional validators:* when your Aurora Chain is deployed, it automatically gets all the Near Blockchain validators, which are above 300.
+2. *Configuration*: create whitelists, choose your own custom token, gas mechanics, integrate KYC, and much more.
+3. *Complete ecosystem*: the Rainbow Bridge, on-ramps, indexers, oracles, DEXes, etc., Aurora Cloud provides a selection of integrations that can be deployed to your chain in order to speed up development
+4. *Focus on your business*: the tech side of the integration and support are entirely on the [Aurora Labs](https://auroralabs.dev/) team's shoulders.
```

Comment:
let's not write this like this. You can say that Aurora Labs team is available to support them in each step of the integration

Reply:
@armanddid 
Like this will be better? Do we need to rename the title?
Maybe to *Reliable Support* or smth alike?

4. *Focus on your business*: [Aurora Labs](https://auroralabs.dev/) team is available to support each step of the integration.

Or maybe we can just remove this one? I mean, the point wasn't about the support, but more about the fact the most work needed to be done - will be done by us or Aurora Cloud automations :)